backend/src/test/kotlin/fr/gouv/cnsp/monitorfish/infrastructure/api/bff/BeaconMalfunctionControllerITests.kt

```diff
@@ -30,7 +29,8 @@ import org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPat
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
 import java.time.ZonedDateTime
 
-@Import(SecurityConfig::class, OIDCProperties::class, SentryConfig::class)
+@Import(SentryConfig::class)
+@AutoConfigureMockMvc(addFilters = false)
```

Comment:
Une question : ça représente quoi exactement les filters ? Je vois ça dans la doc mais je ne sais pas trop à quoi ça correspond : _If filters from the application context should be registered with MockMVC._.

Reply:
Vu en visio